microcosm_flask/tests/conventions/fixtures.py

```diff
@@ -46,8 +47,8 @@ class NewPersonBatchSchema(Schema):
 
 
 class UpdatePersonSchema(Schema):
-    firstName = fields.Str(attribute="first_name")
-    lastName = fields.Str(attribute="last_name")
+    firstName = fields.String(attribute="first_name")
+    last_name = fields.String(data_key="lastName")
```

Comment:
I'm guessing the test assertions don't need to be updated because the output is exactly the same?

Reply:
Indeed. The end swagger shouldn't have any changes.